manifests/init.pp

```diff
@@ -89,6 +90,7 @@
     create_resources('firewalld_zone',      $zones)
     create_resources('firewalld_service',   $services)
     create_resources('firewalld_rich_rule', $rich_rules)
+    create_resources('firewalld::custom_service', hiera('firewalld::custom_service', $custom_services))
```

Comment:
Why do you have a hiera lookup for firewalld::custom_service here?

I think this should be

```
create_resources('firewalld::custom_service', $custom_services)
```

?